packages/plugin-auto-nav-sidebar/src/walk.ts

```diff
@@ -32,7 +34,15 @@ export async function scanSideMeta(
     sideMeta = (await fs.readJSON(metaFile, 'utf8')) as SideMeta;
   } catch (e) {
     // If the `_meta.json` file doesn't exist, we will generate the sidebar config from the directory structure.
-    const subItems = await fs.readdir(workDir);
+
+    const includedFiles = globby.sync('**', {
```

Comment:
Here `globby.sync` will traverse all the files and subdirectories under the directory but we only want to traverse the first level. You can change the params as follows:

```js
globby.sync('*', {
  cwd: workDir,
  ignore: excludedFiles ?? [],
  expandDirectories: false,
  onlyFiles: false,
});
```

Reply:
@Tyneor

Reply:
Hello @sanyuan0704

This doesn't work for the use case `assets/**/*` I used in my original issue. With `assets/**/*` it is necessary to match the files under the directory.

I will still add `expandDirectories: false` + my second method because I think users excluding `assets/**/*` will be fine excluding `assets/**`